services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/security/AccessRules.kt

```diff
@@ -26,6 +26,11 @@ object AccessRules {
                 "/v3/api-docs/**",
             )
             .permitAll()
+            .mvcMatchers(HttpMethod.POST, "/testdata/unlink-applications")
```

Comment:
Would it be better to require an access token and log the user doing this operation? I don't have a clear opinion myself so I'm just pondering.

Reply:
Requiring an access token would in practice limit the use to developers.